### ISSUE TITLE
[Bug] PineconeIndexOperationClient BASE_URL_SUFFIX incorrect

### ISSUE THREAD

**Issue by JackMatthewRimmer** [pinecone-io/pinecone-java-client#56]
### Is this a new bug?

- [X] I believe this is a new bug
- [X] I have searched the existing issues, and I could not find an existing issue for this bug

### Current Behavior

are PineconeNotFoundException is thrown when calling list indexes, reason being that 
    public static final String BASE_URL_SUFFIX = ".pinecone.io/databases/";
contains a slash at the end where as in the docs the url does not contain a slash example being...
https://controller.us-east1-gcp.pinecone.io/databases

### Expected Behavior

Should be able to make a request to the correct url

### Steps To Reproduce

call list indexes with correct details

### Relevant log output

```shell
io.pinecone.exceptions.PineconeNotFoundException: 
        at io.pinecone.exceptions.HttpErrorMapper.mapHttpStatusError(HttpErrorMapper.java:13)
```

### Environment

```markdown
- Mac OS:
- 17:
- 0.7.2:
```

**Comment by rohanshah18** [comment]
As I mentioned in the [previous post](#54), this issue is going to be tracked by [the already opened issue](https://github.com/pinecone-io/pinecone-java-client/issues/54#issuecomment-1875480750). Thanks!

**Comment by rohanshah18** [comment]
@JackMatthewRimmer I have released [Java SDK v0.7.4](https://github.com/pinecone-io/pinecone-java-client/releases/tag/v0.7.4), please let me know if the new release resolves the gcp-starter env issues.

**Comment by JackMatthewRimmer** [comment]
@rohanshah18 is all working now 😄
